src/pfe/file-watcher/server/src/utils/workspaceSettings.ts

```diff
@@ -190,130 +210,153 @@ export async function getWorkspaceSettingsInfo(): Promise<any> {
-        dockerPushEC = dockerPush.exitCode;
-        logger.logInfo("testDeploymentRegistry exit code: " + dockerPushEC);
+        buildahPush = (await processManager.spawnDetachedAsync("imagePushRegistryTest1", "buildah", ["push", "--tls-verify=false", pullImage, `${imagePushRegistry}/${pullImage}`], {}));
+        buildahPushEC = buildahPush.exitCode;
+        logger.logDebug("testImagePushRegistry exit code: " + buildahPushEC);
     } catch (err) {
-        const msg = `Codewind was unable to push the ${pullImage} image to the Deployment Registry ${deploymentRegistry}/${pullImage}. Please make sure it is a valid Deployment Registry with the appropriate permissions.`;
+        const msg = `Codewind was unable to push the ${pullImage} image to the Image Push Registry ${imagePushRegistry}/${pullImage}. Please make sure it is a valid Image Push Registry with the appropriate permissions.`;
         logger.logError(msg);
         logger.logError(err);
 
         const data: any = {
-            deploymentRegistryTest: deploymentRegistryTest,
+            imagePushRegistryTest: imagePushRegistryTest,
             msg: msg
         };
-        io.emitOnListener("deploymentRegistryStatus", data);
+        io.emitOnListener("imagePushRegistryStatus", data);
 
-        return { "statusCode": 200, "deploymentRegistryTest": deploymentRegistryTest, "msg": msg};
+        return { "statusCode": 200, "imagePushRegistryTest": imagePushRegistryTest, "msg": msg};
     }
 
-    if (dockerPushEC == 0) {
-        deploymentRegistryTest = true;
+    if (buildahPushEC == 0) {
+        imagePushRegistryTest = true;
     }
 
-    logger.logInfo("Deployment Registry Test Status: " + deploymentRegistryTest);
+    logger.logInfo("Image Push RegistryTest Test Status: " + imagePushRegistryTest);
     let msg;
-    if (deploymentRegistryTest) {
-        msg = "Codewind projects on Kubernetes will build with the Deployment Registry: " + deploymentRegistry;
-        logger.logInfo("Successful Deployment Registry test");
+    if (imagePushRegistryTest) {
+        msg = "Codewind projects on Kubernetes will build with the Image Push Registry: " + imagePushRegistry;
+        logger.logInfo("Successful Image Push Registry test");
         logger.logInfo(msg);
     } else {
-        msg = "Codewind projects on Kubernetes cannot build with the Deployment Registry, as Codewind encountered an issue when pushing an image to: " + deploymentRegistry + ". Please make sure it is a valid Deployment Registry with the appropriate permissions.";
-        logger.logInfo("Un-successful Deployment Registry test");
-        logger.logInfo(msg);
+        msg = "Codewind projects on Kubernetes cannot build with the Image Push Registry, as Codewind encountered an issue when pushing an image to: " + imagePushRegistry + ". Please make sure it is a valid Image Push Registry with the appropriate permissions.";
```

Comment:
This message sounds a bit awkward. Suggest updating to
`Codewind cannot build projects with the current Image Push Registry. The image could not be pushed to: " + imagePushRegistry + ". Please make sure it is a valid Image Push Registry with the appropriate permissions.`

Reply:
updated